solutions/observability/get-started/quickstart-elastic-cloud-otel-endpoint.md

```diff
@@ -100,22 +100,28 @@ To send data to the {{motlp}} from the {{edot}} Collector or the contrib Collect
 ```yaml
 exporters:
   otlp:
-    endpoint: https://<motlp-endpoint>
+    endpoint: https://<motlp-endpoint> <1>
     headers:
-      Authorization: ApiKey <your-api-key>
+      Authorization: ApiKey <your-api-key> <2>
 ```
 
+1. See [Step 2](#locate-your-motlp)
+2. See [Step 3](#create-an-api-key)
```

Comment:
```suggestion
1. The endpoint retrieved at [step 2](#locate-your-motlp)
2. The API key created at [step 3](#create-an-api-key)
```
We should avoid "see"/"view" in our language.

Same comment for other occurrences in this PR

Reply:
https://github.com/elastic/docs-content/pull/2849/commits/13d4b5b24dad678be347f5a179ffebef2666baef